src/pages/search/index.js

```diff
@@ -73,33 +61,22 @@ export const Search = ({
   }, [])
 
   useEffect(() => {
-    if (facets) {
-      const facetNames = formatFacetNames(Object.keys(facets))
-
-      setConfig({
-        filterOptions: facetNames
-      })
-
-      if (query) {
-        setSearch({
-          ...getSearchQueryParam(query)
-        })
-      }
+    if (facets) setFacetNames(formatFacetNames(Object.keys(facets)))
+    if (query) {
+      setSearch(query)
+      setUserSearchTerm(search) // resets previous input value
+      gtag.trackSearchQuery(query)
     }
   }, [facets, query])
 
-  useEffect(() => {
-    gtag.trackSearchQuery(query)
-  }, [query])
-
   // TODO: Remove when refactring search in a future issue
   if (!isPageReady) return <Spinner />
 
   return (
     <>
-      <PageTitle title={`${query.search ? query.search : ''} Results -`} />
+      <PageTitle title={`${search || ''} Results -`} />
```

Comment:
```suggestion
      <PageTitle title={`${search} Results -`} />
```

There is already a default empty string for this so we can assume it is defined.